third_party/blink/renderer/modules/ml/model.cpp

```diff
@@ -64,12 +64,22 @@ bool InvalidOperand(const OperandOptions* options,
       if (!options->hasDimensions())
         return InvalidState("Data type is invalid.", exception_state);
       break;
+    case NeuralNetworkContext::kTensorQuant8SymmPerChannel:
```

Comment:
Could you please refer to https://github.com/intel/webml-polyfill/blob/master/src/nn/Model.js#L225 for validation?

Reply:
Yes, I add the validation code into the `setOperandSymmPerChannelQuantParams` method.